.eslintrc.json

```diff
@@ -5,6 +5,7 @@
         "commonjs": true
     },
     "rules": {
+        "no-unused-vars": ["error", {"vars": "all", "args": "none"}],
```

Comment:
What do you think, @albertinad and @MSLaguana? Personally I don't like having a restriction against unused parameters, because sometimes they're useful (particularly on callback functions -  makes it clearer in the code the parameters the functions the code will be called with). Our jshint config allowed for this, but not our eslint config.